docs/calicoctl/node.md

```diff
@@ -71,7 +74,9 @@ Options:
 
 ### calicoctl node 
 
-This command starts a container using the `calico/node` Docker image. 
+This command performs two actions:
+1. Initialize this host for calico by setting first-time config in etcd.
+2. Start the `calico/node` Docker container.
```

Comment:
(Optional) Start Calico services.  By default (or via `--runtime=docker`) the command does this by downloading the `calico/node` Docker image and running it in a container.